notebooks/en/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -113,18 +113,35 @@
    "id": "1e680f3d-4900-46cc-8b49-bb6ba3e27e2b",
```

Comment:
<p>"Inference Endpoints offers a number of GPUs that you can choose from."</p><p> </p><p>You can use a nice Tip callout here as well :)</p><p> </p><p>"&gt; [!TIP]</p><p> &gt; You may need to email us for access to some architectures."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>